### PR TITLE
Create WETH Page

### PR DESCRIPTION
**Pull request by fabioberger**
This PR:
* Adds a permanent WETH page to Portal
* Removes the "Convert" button from the balances page
* Re-designs the conversion dialog to be specifically a wrap/unwrap dialog
* Uses a new WETH icon
* Refactors tokenAmountInput so it can be displayed with or without a label
* Adds the ability to clear trackedTokens local storage since we must force users to start tracking the new WETH and stop tracking the old.

screen shot 2017-12-14 at 10 58 24

screen shot 2017-12-14 at 10 58 33

### CODE REVIEW

File: packages/website/ts/components/eth_weth_conversion_button.tsx [hunk 1]
```diff
@@ -12,12 +12,15 @@ import {errorReporter} from 'ts/utils/error_reporter';
...
 interface EthWethConversionButtonProps {
+    direction: Side;
     ethToken: Token;
     ethTokenState: TokenState;
     dispatcher: Dispatcher;
     blockchain: Blockchain;
     userEtherBalance: BigNumber;
-    onError: () => void;
+    isOutdatedWrappedEther: boolean;
+    onConversionSuccessful?: () => void;
```
Comment (LogvinovLeon) [inline packages/website/ts/components/eth_weth_conversion_button.tsx]: IfExists

File: packages/website/ts/components/inputs/token_amount_input.tsx [hunk 2]
```diff
@@ -8,9 +8,9 @@ import {BalanceBoundedInput} from 'ts/components/inputs/balance_bounded_input';
 import {InputErrMsg, Token, TokenState, ValidatedBigNumberCallback, WebsitePaths} from 'ts/types';
 
 interface TokenAmountInputProps {
-    label: string;
     token: Token;
     tokenState: TokenState;
+    label?: string;
```
Comment (LogvinovLeon) [inline packages/website/ts/components/inputs/token_amount_input.tsx]: IfExists

File: packages/website/ts/components/portal_menu.tsx [hunk 3]
```diff
@@ -50,6 +50,14 @@ export class PortalMenu extends React.Component<PortalMenuProps, PortalMenuState
                 >
                     {this.renderMenuItemWithIcon('Trade history', 'zmdi-format-list-bulleted')}
                 </MenuItem>
+                <MenuItem
+                    style={this.props.menuItemStyle}
+                    className="py2"
+                    to={`${WebsitePaths.Portal}/weth`}
+                    onClick={this.props.onClick.bind(this)}
+                >
+                    {this.renderMenuItemWithIcon('ETH wrapper', 'zmdi-circle-o')}
```
Comment (abandeali1) [inline packages/website/ts/components/portal_menu.tsx]: Prefer `Wrapped ETH` for consistency.

Comment (fabioberger) [reply packages/website/ts/components/portal_menu.tsx]: Hm... This is the menu item name. It should be a call to action similar to the others: `Generate order`, `Fill order`...

File: packages/website/ts/components/eth_weth_conversion_button.tsx (hunk 1 above)
Comment (fabioberger) [reply packages/website/ts/components/eth_weth_conversion_button.tsx]: I don't think it makes sense to add the `IfExists` prefix to prop names. You might or might not define it when instantiating the component. I'll add a defaultProp value instead.

**Pull request merged**